### PR TITLE
[Teams] Google Workspace Tutorial

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

File: products/cloudflare-one/src/content/tutorials/google-workspace-saas.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+This tutorial covers how to use Cloudflare as a single sign on provider by setting up Access for SaaS.
+
+**⏲️ Time to complete:** 15 minutes
+
+1. In your Google Workspace account, create an SSO third-party identity provider.
+
+    ![SSO profile on Google Workspace](../static/zero-trust-security/google-workspace-saas/sso-profile.png)
+
+1. Check the **Setup SSO with a third party identity provider** option.
+1. On the Teams Dashboard, navigate to **Access** > **Applications** and create a SaaS application.
```
Comment (kennyj42) [inline products/cloudflare-one/src/content/tutorials/google-workspace-saas.md:24]: This isn't really in your Workspace account. Not sure if we should split it out?

**Review by kennyj42 — APPROVED** [review]

File: products/cloudflare-one/src/content/tutorials/google-workspace-saas.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+**⏲️ Time to complete:** 15 minutes
+
+
+## Create an SSO provider in Google Workspace
+
+1. In your Google Workspace account, create an SSO third-party identity provider.
+
+    ![SSO profile on Google Workspace](../static/zero-trust-security/google-workspace-saas/sso-profile.png)
+
+1. Check the **Setup SSO with a third party identity provider** option.
```
Comment (kodster28) [inline products/cloudflare-one/src/content/tutorials/google-workspace-saas.md]:
```suggestion
1. Select **Setup SSO with a third party identity provider**.
```

File: products/cloudflare-one/src/content/tutorials/google-workspace-saas.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+1. Click **Next** to create an Access policy and use the following mapping to set up your Workspace sign-in:
+
+    | Google Workspace value | Cloudflare value |
+    | ----- | ---- |
+    | Sign-in page URL | SSO Endpoint |
+    | Sign-out page URL | SSO Endpoint |
+    | Verification Certificate | Public Key |
+
+1. Next, you can choose to check **Use a domain specific issuer**. If you check this option, Google will send an issuer specific to your domain (where `your_domain.com` is replaced with your actual primary Google Workspace domain name).
```
Comment (kodster28) [inline products/cloudflare-one/src/content/tutorials/google-workspace-saas.md]:
```suggestion
1. Next, you can select **Use a domain specific issuer**. If you select this option, Google will send an issuer specific to your domain (where `your_domain.com` is replaced with your actual primary Google Workspace domain name).
```

File: products/cloudflare-one/src/content/tutorials/google-workspace-saas.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+## Test the integration
+
+You can now test the integration by going to `https://google.com/a/<yourdomain>`.
+
+## Troubleshooting
+
+`Error: “G Suite - This account cannot be accessed because the login credentials could not be verified.”`
+
+If you see this error, it is likely that the Public Key and Private Key do not match. Double-check that your certificate file matches your Public Key.
```
Comment (kodster28) [inline products/cloudflare-one/src/content/tutorials/google-workspace-saas.md]:
```suggestion
If you see this error, it is likely that the Public Key and Private Key do not match. Confirm your certificate file matches your Public Key.
```

**Review by kodster28 — APPROVED** [review]

**Pull request merged**